src/activity_actor.cpp

```diff
@@ -1339,6 +1345,65 @@ std::unique_ptr<activity_actor> lockpick_activity_actor::deserialize( JsonIn &js
     return actor.clone();
 }
 
+void ebooksave_activity_actor::start( player_activity &act, Character &/*who*/ )
+{
+    const int pages = pages_in_book( book->typeId() );
+    const time_duration scanning_time = pages < 1 ? time_per_page : pages * time_per_page;
+    add_msg_debug( "ebooksave pages = %d", pages );
+    add_msg_debug( "shearing_time time = %s", to_string( scanning_time ) );
```

Comment:
```suggestion
    add_msg_debug( "scanning_time time = %s", to_string( scanning_time ) );
```

Reply:
Oops. I messed up on the copy and paste. Thanks.